client/src/layout/MobileMenu/MobileMenu.scss

```diff
@@ -0,0 +1,34 @@
+.mobile-menu-container.active{
+  display: grid;
+  align-items: center;
+  width: 100%;
+  height: 90%;
+  background-color: darkgray;
+  transition: 0.3s ease-in-out;
+}
+.mobile-menu-container{
```

Comment:
now that we are leveraging `scss`, let's make the most of it by consolidating these classes with something like 
```js
.mobile-menu-container {
display: none;

::active {
 display: grid;
  align-items: center;
  width: 100%;
  height: 90%;
  background-color: darkgray;
  transition: 0.3s ease-in-out;
  }
}
```